Jenkinsfile

```diff
@@ -4,7 +4,7 @@ getApproval()
 
 pipeline {
   agent {
-    label 'x86_64&&macOS && !macOS_10_15'  // xdoc doesn't work on Catalina
+    label 'x86_64 && macOS && !macOS_10_15'  // macOS handles multiprocessing in python differently causing causing runime issues
```

Comment:
bad document change?

Reply:
the comment was part of the commit moving it to linux, I then changed python to 3.7.6 and temporeraly back to macOS. It now passes on macOS (so multiprocessing issue is with 3.10) and I've now removed the comment and changed moved the testing to linux